content/posts/Vivek_Kumar_GSoC2025_FinalReport.md

```
@@ -0,0 +1,165 @@
+
+### Resizable & draggable view of tools window
+
+reziable 
+
+### Verilog feature documentation
+
+As part of this project, one of the primary tasks has been to document the verilog feature in deatil with examples for the CircuitVerse users. I have created detailed documentation for both the Verilog feature with examples for the CircuitVerse users to experiment with it and use them.
+
+A special focus was also given to creating the documentation for the developers of the CircuitVerse so that they can easily setup the Verilog feature in thier local machine and develop it further.
+
+##### Documenting the Verilog Features:
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)